app/src/utils/api.ts

```diff
@@ -1,20 +1,21 @@
-const API_HOST = process.env.VITE_BACKEND_URL || 'https://api.dun.wtf'
-
-const headers = {
-  'Content-Type': 'application/json',
-  Authorization: `Bearer ${localStorage.getItem('token')}`,
-}
+const API_HOST = process.env.VITE_BACKEND_URL || 'https://api.dun.wtf';
 
 export async function apiRequest<T>(url: string, options: RequestInit = {}): Promise<T> {
   try {
-    const res = await fetch(API_HOST + '/api/' + url, { headers, ...options })
-    if(!res.ok) {
-      throw new Error('HTTP error, status = ' + res.status)
+    const headers = {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${localStorage.getItem('token') || ''}`,
+    };

```

Comment:
It's better to declaring `headers ` in the global scope of the file (on second line for ex).